docs/okeps/okep-5094-layer2-transit-router.md

```diff
@@ -0,0 +1,870 @@
+    
+    subgraph node1["node1 <b>(updated)</b>"]
+        subgraph GR-node1
+            rtotr-GR-node1["trtor-GR-node1 
+            100.65.0.2/16 100.88.0.6/30 (0a:58:64:41:00:02)"]
+        end
+        
+        subgraph layer2-switch["layer2-switch"]
+            stor-ovn_cluster_router["stor-ovn_cluster_router 
+                type: router"]
+            stor-GR-node3["stor-GR-node3  100.65.0.4/16 (0a:58:64:41:00:04)
+                type: remote"]
+        end
+        
+        
+        
+        subgraph ovn_cluster_router["ovn_cluster_router "]
+            trtor-GR-node2["trtor-GR-node2 100.88.0.13/30"]
+            trtor-GR-node1["trtor-GR-node1 100.88.0.5/30"]
+            rtos-layer2-switch["rtos-layer2-switch 203.203.0.1/24 <b>100.65.0.2/16</b> (0a:58:CB:CB:00:01)"]
+        end
+    end
+    
+    subgraph node2["node2 <b>(updated)</b>"]
+        subgraph GR-node2
+            rtotr-GR-node2["rtotr-GR-node2 100.65.0.3/16 100.88.0.14/30 (0a:58:64:41:00:03)"]
+        end
+    end
+    
+    subgraph node3["node3 <b>(old topo)</b>"]
+        subgraph VM["Virtual Machine"]
+            class VM vmStyle;
+            term["default gw 
+            203.203.0.1
+            (0a:58:64:41:00:04)"]
+        end
+        subgraph GR-node3
+            rtos-GR-node3["rtos-GR-node3  203.203.0.1/24 100.65.0.4/16 (0a:58:64:41:00:04)"]
+        end
+        subgraph layer2-switch-node3["layer2-switch"]
+            stor-GR-node1["stor-GR-node1
+                100.65.0.2/16 (0a:58:64:41:00:02) 
+                type: remote"]
+            stor-GR-node3-local["stor-GR-node3 
+                type: router"]
+        end
+    end
+    
+    rtotr-GR-node1 <--> trtor-GR-node1
+    rtotr-GR-node2 <--> trtor-GR-node2
+    rtos-GR-node3 <--> stor-GR-node3-local
+    stor-GR-node3-local <--> stor-GR-node3
+    rtos-layer2-switch <--> stor-ovn_cluster_router
+    stor-GR-node1 <--> stor-ovn_cluster_router
+    VM <--> layer2-switch-node3
+    
+    class VM vmStyle;
+    class rtotr-GR-node1 portStyle;
+    class rtotr-GR-node2 portStyle;
+    class rtotr-GR-node3 portStyle;
+    class trtor-GR-node1 portStyle;
+    class trtor-GR-node2 portStyle;
+    class trtor-GR-node3 portStyle;
+    class stor-GR-node3 portStyle;
+    class rtos-GR-node3 portStyle;
+    class stor-ovn_cluster_router portStyle;
+    class rtos-layer2-switch portStyle;
+    class stor-GR-node3-local portStyle;
+    class stor-GR-node1 portStyle;
+    class GR-node1 routerStyle;
+    class GR-node2 routerStyle;
+    class GR-node3 routerStyle;
+    class ovn_cluster_router routerStyle;
+    class layer2-switch switchStyle;
+    class layer2-switch-node3 switchStyle;
+    class term termStyle;
+    class node1,node2,node3 nodeStyle;
+```
+
+With this topology updated nodes should keep the remote node connection towards
+not updated nodes using the original switch remote LSPs, since the old nodes
+has not transit router yet and after detecting that the node has the new topology
```

Comment:
you mean the updated nodes should keep the remote node connection towards the non-updated nodes using the remote GR-<non-updated node> port as part of its layer2-switch?

This paragraph is kind of ambiguous. Let's call out specifically what components in OVN we are talking about and what will still exist. You can use the diagram to provide examples.

Reply:
> you mean the updated nodes should keep the remote node connection towards the non-updated nodes using the remote GR- port as part of its layer2-switch?
> 

Yes, since those not updated remote nodes do not has the transit router yet they only connection to other nodes is with the remote LSP

> This paragraph is kind of ambiguous. Let's call out specifically what components in OVN we are talking about and what will still exist. You can use the diagram to provide examples.

I will fill the "TODO" diagram I have above this with the topolog of upgraded and not upgraded node connected.